utils/api/requests.js

```diff
@@ -72,3 +72,100 @@ export const sendMessage = ({ id, message, files }) => {
 
   posting(`/quote_groups/${id}/notes.json`, note)
 }
+
+export const useInitializeRequest = (id) => {
+  const { data, error } = useSWR(`/wares/${id}/quote_groups.json`, fetcher)
+  let dynamicForm = { name: data?.name }
+
+  if (data?.dynamic_form) {
+    const defaultSchema = data.dynamic_form.schema
+    const defaultOptions = data.dynamic_form.options
+    const schema = dynamicFormSchema(defaultSchema)
+
+    dynamicForm = {
+      ...dynamicForm,
+      schema,
+      uiSchema: dynamicFormUiSchema(schema, defaultOptions),
+    }
+  }
+
+  return {
+    dynamicForm,
+    isLoadingInitialRequest: !error && !dynamicForm,
+    isInitialRequestError: error,
+  }
+}
+
+// TODO(alishaevn): https://github.com/assaydepot...scientist_api_v2/app/serializers/scientist_api_v2/dynamic_form_serializer.rb#L39
+// update the method at the code above to return the configured schema below
+export const dynamicFormSchema = (defaultSchema) => {
+  // TODO(alishaevn): may need to account for multiple forms
+
+  const removedProperties = [
+    'concierge_support', 'suppliers_identified', 'price_comparison', 'number_suppliers',
+    'supplier_criteria', 'supplier_confirmation'
+  ]
```

Comment:
in the webstore we are focused on a single supplier and the mvp doesn't have concierge features, so we don't need these.